docs/guide/advanced/accessibility.md

```diff
@@ -117,6 +116,36 @@ contained icon.
 
 ![](../../images/a11y/alttext-iconbuttons.svg?raw=true)
 
+## A note on `aria-label`
+
+Although you could provide accessible labels to your elements via the `aria-label` attribute, we
+generally suggest using your chosen CSS framework's "visually hidden" utility instead whenever
+possible, since some assistive technology fails to announce `aria-label` attributes in case the
+element is not of an appropriate semantic role.
```

Comment:
The main reason I don't like `aria-label` is that it gets exposed by all major browsers as AXDescription while normal content is exposed as AXTitle, and according to the spec, AXDescription should be read out in the system language, meaning your `lang=…` attribute gets ignored and your screen reader might sound funky.

Reply:
Neato, I've updated this description and removed all traces of `aria-label` from the illustrations.